octoprint_mrbeam/templates/laserhead_changed.jinja2

```diff
@@ -1,19 +1,87 @@
+            <p> {{ _("Every device older than the dreamcut row will not support the new and advanced Laser head properly. There are just too many improvements that are missing in the earlier states of our model rows like Mr Beam and Mr Beam II. Please contact our support tor further steps.") }}</p>
         </div>
     </div>
+    {# Not supported laserhead  - END #}
+
+
+    {# Steps for laserhead S  - START #}
+    <div class="modal-body" data-bind="visible: show_step_for_laserhead_s(1)">
+        <div>
+            <img loading="lazy" src="/plugin/mrbeam/static/img/Welcome.png">
+            <h4> {{ _("Your Mr Beam dreamcut [S] Laser head has been successfully installed.") }}</h4>
+            <p> {{ _("With this Laser head you can work now even faster.") }}</p>
+            <p> {{ _("This means that we also update the default material settings to the new laser head as long it is installed to provide you the best possible support. However, we do not update your custom materials, this will need your personal testing and finetuning.") }}</p>
+            <p> {{ _("Read about the changes you can expect with the MrBeam II dreamcut [S] Laser head.") }} <a
+                    target="_blank"
+                    href="{{ _("https://support.mr-beam.org/en") }}/support/solutions/articles/43000640531">
+                » {{ _("Read more") }} </a></p>
+        </div>
+    </div>
+    {# Steps for laserhead S  - END #}
+
+    {# Steps for laserhead x  - START #}
+    <div class="modal-body" data-bind="visible: show_step_for_laserhead_x(1)">
+        <div>
+            <img loading="lazy" src="/plugin/mrbeam/static/img/Welcome.png">
+            <h4> {{ _("Your Mr Beam II dreamcut [X] laser head has been successfully connected.") }}</h4>
+            <p> {{ _(" You can now work even faster with this laser head.") }}</p>
+            <p> {{ _("We have adapted the standard material settings to the new laser head so that you can start directly. Your personal material settings remain unchanged. To adjust them to the increased performance of the laser head, you must perform tests yourself and save the results.") }}</p>
+            <p> {{ _("All the new features that await you with the dreamcut [X] laser head can be found in this Knowledgebase article:") }} <a
+                    target="_blank"
+                    href="{{ _("https://support.mr-beam.org/en") }}/support/solutions/articles/43000693865">
+                » {{ _("Learn more") }} </a></p>
+        </div>
+    </div>
+
+    <div class="modal-body" data-bind="visible: show_step_for_laserhead_x(2)">
+        <div>
+            <p><img loading="lazy" src="/plugin/mrbeam/static/img/speed_improvement.png"></p>
+            <h4> {{ _("Your Mr Beam is even faster now!") }}</h4>
+            <ul>
+                <li> {{ _("If you had a Mr Beam II dreamcut before, you can now cut up to four times as fast.") }}</li>
```

Comment:
I think this is the correct way, but please double check
```suggestion
                <li> {{ _("If you had a Mr Beam II dreamcut before, you can now cut up to four times faster.") }}</li>
```

Reply:
Deepl also suggest as fast